test/RootStateSchema.json

```diff
@@ -4273,6 +4276,9 @@
                 "isCoreToken": {
                     "type": "boolean"
                 },
+                "isNative": {
+                    "type": "boolean"
```

Comment:
I think isNative is optional

Reply:
It is optional -- it's marked as such in the `BaseToken` type.